src/interfaces/assistants_web/src/cohere-client/generated/types.gen.ts

```diff
@@ -299,17 +303,8 @@ export type File = {
   created_at: string;
   updated_at: string;
   user_id: string;
-  conversation_id: string;
-  file_name: string;
-  file_path: string;
-  file_size?: number;
-};
-
-export type FilePublic = {
-  id: string;
-  created_at: string;
-  updated_at: string;
-  conversation_id: string;
+  conversation_id?: string | null;
```

Comment:
why is this optional now?

Reply:
ohh we are sharing the interface for Agent and Conversation files. Could we separate them?

Reply:
@knajjars yep good point, lemme change and push here

Reply:
@scott-cohere let me know if you need help to update the FE with your changes

Reply:
made the changes to split Files schema into conversation vs. agent files. changes should be reflected in
`src/interfaces/assistants_web/src/cohere-client/generated/types.gen.ts`

@knajjars @tomtobac